docs/sources/configuration/integrations/azure-exporter-config.md

```diff
@@ -0,0 +1,137 @@
+---
+aliases:
+- /docs/agent/latest/configuration/integrations/azure-exporter-config/
+title: azure_exporter_config
+---
+
+# azure_exporter_config
+
+## Overview
+The `azure_exporter_config` block configures the `azure_exporter` integration, which is an embedded version of
+[`azure-metrics-exporter`](https://github.com/webdevops/azure-metrics-exporter). This allows for
+the collection of metrics from [Azure Monitor](https://azure.microsoft.com/en-us/products/monitor). The
+exporter uses [Azure Resource Graph](https://azure.microsoft.com/en-us/get-started/azure-portal/resource-graph/#overview) 
+queries to identify resources for gathering metrics. 
+
+The exporter supports all metrics defined by [Azure Monitor](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/metrics-supported).
+Metrics for this integration are exposed with the template `azure_{type}_{metric}_{aggregation}_{unit}`. As an example
+the Egress metric for BlobService would be exported as `azure_microsoft_storage_storageaccounts_blobservices_egress_total_bytes`.
+
+## Authentication
+
+The agent will need to be running in an environment which has access to azure. The exporter uses the Azure SDK for go and supports
+providing authentication via https://learn.microsoft.com/en-us/azure/developer/go/azure-sdk-authentication?tabs=bash#2-authenticate-with-azure.
+
+The account used by the agent will need,
+* [Read access to the resources which will be queried by Resource Graph](https://learn.microsoft.com/en-us/azure/governance/resource-graph/overview#permissions-in-azure-resource-graph)
+* Permissions to call the [Microsoft.Insights Metrics API](https://learn.microsoft.com/en-us/rest/api/monitor/metrics/list) which should be the `Microsoft.Insights/Metrics/Read` permission
+
+## Configuration options:
```

Comment:
If you have something at hand, it'd be nice to have in the docs a simple example of how to scrape some Azure metrics, for example, some full config for scraping some blob storage metrics, or azure compute instances

Reply:
👍 I'll add some sample configs above the config options